templating-engine/src/index.js

```diff
@@ -2,12 +2,14 @@ const express = require('express');
 
 const app = express();
 
+app.set('view engine', 'ejs');
+
 const languages = ['JavaScript', 'Haskell', 'Python'];
 
-app.get('/', (req, res) => {
-  return res.send(`Hello ${languages[0]}`); // Use res.render() to render the ejs file instead of sending text response
-});
+// Index Page
+app.get('/', (req, res) => res.render('../views/pages/index', { languages }));
 
-app.listen(3000, () => {
-  console.log('Server is listening on port 3000');
-});
+const port = process.env.PORT || 3000;
+app.listen(port, () => {
+  console.log(`Server is listening on port ${port}`);
+});
```

Comment:
@prateekinstillion , I think it's a good practice to have new lines at the end of files. What do you say? 
There is one article explaining the issue here: https://evanhahn.com/newline-necessary-at-the-end-of-javascript-files/

Reply:
Thanks @abi18av noted.

Reply:
Thanks @abhi18av noted.